perfeed/models/pr_summary.py

```diff
@@ -65,3 +65,14 @@ class PRSummary(BaseModel):
     @classmethod
     def to_json_schema(cls) -> str:
         return json.dumps(cls.model_json_schema(), indent=2)
+
+
+class PRSummaryMetadata(BaseModel):
```

Comment:
What's the purpose of this dataclass? And should `metadata` be a part of `PRSummary` instead of a standalone class?

Reply:
the way `PRSummary` is designed to be not only a data class and but also as a prompt. If we add metatada info there, you are asking llm to repeat some info in ur prompt, which incurs cost and info fidelity issues.

Reply:
OK, please document the purpose of this class